packages/babel-plugin-transform-async-to-promises/src/index.js

```diff
@@ -42,11 +42,12 @@ function transformAsyncToPromises(api, options) {
         }; 
 
   if (options.codeGenerationOptions) {
-      let keys = Object.keys(options.codeGenerationOptions);
-      for (let i=0; i<keys.length;i++) {
-	  if (keys[i] in opts)
+    let keys = Object.keys(options.codeGenerationOptions);
+    for (let i=0; i<keys.length;i++) {
+	    if (keys[i] in opts) {
```

Comment:
this still needs one more space, so that it's two spaces indented from line 46

Reply:
I can't see it in the online editor (it looks like there are two spaces) - can you fix it?

Reply:
I can't; this is a PR to your fork (from your fork), and I don't have write access there.